backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/FeatureSupported.java

```diff
@@ -431,4 +431,14 @@ public static boolean isConvertDiskSupported(VDS vds) {
     public static boolean isHybridBackupSupported(VM vm) {
         return vm != null && supportedInConfig(ConfigValues.UseHybridBackup, vm.getCompatibilityVersion());
     }
+
+    /**
+     * Checks if the VM can be set with dedicated CpuPinningPolicy.
+     *
+     * @param version Compatibility version to check for.
+     * @return true if the VM is capable to be set with dedicated policy.
+     */
+    public static boolean isDedicatePolicySupported(Version version) {
+        return Version.v4_7.lessOrEquals(version);
```

Comment:
we better replace it with a config in vdc_options so users can disable it, if needed